src/domain/ContentPage/ContentPage.module.scss

```diff
@@ -62,4 +62,5 @@
 
 .listView {
   padding: 0 unit(1);
+  margin-bottom: 100px;
```

Comment:
You don't have to change this, but large fixed margins are usually problematic, maybe percentages or even `em`s might be better.

Reply:
This is a bit questionable indeed, but the simplest fix for not having the `position: sticky` category navigation float on top of the list on safari, which has a height of 100px.

Reply:
Fair enough.